packages/nextjs/src/config/webpack.ts

```diff
@@ -27,6 +27,13 @@ export { SentryWebpackPlugin };
 // TODO: merge default SentryWebpackPlugin include with their SentryWebpackPlugin include
 // TODO: drop merged keys from override check? `includeDefaults` option?
 
+// In order to make sure that build-time code isn't getting bundled in runtime bundles (specifically, in the serverless
+// functions into which nextjs converts a user's routes), we exclude this module (and all of its dependencies) from the
+// nft file manifests nextjs generates. (See the code dealing with the `TraceEntryPointsPlugin` below.) So that we don't
+// crash people, we therefore need to make sure nothing tries to either require this file or import from it. Setting
+// this env variable allows us to test whether or not that's happened.
+process.env.SENTRY_WEBPACK_MODULE_LOADED = 'true';
```

Comment:
I'd rather just set a global variable here, then we can also use a boolean instead of a string.

Setting on `process.env` has some behaviours that maybe someone might not realize, on windows things become case insensitive (`process.env.FOO` === `process.env.foo`) and everything on `process.env` gets coerced to strings when the getter is called.

Since we don't need the above two behaviours, I would rather just set a mutable global. More obvious what is happening to users, and also allows us to lift the comment up from the usage of the variable to the declaration of the variable (cleaning up the function scope a little).

Reply:
Considering Abhi I changed it to be a global in [728562b](https://github.com/getsentry/sentry-javascript/pull/6267/commits/728562b0f95e88c3ea2bdc67fd124244ec1ba5cf)